packages/chevrotain/src/parse/grammar/gast/gast_public.ts

```diff
@@ -350,6 +365,10 @@ export function serializeProduction(node: IProduction): ISerializedGast {
       idx: node.idx
     }
 
+    if (node.label !== undefined) {
```

Comment:
- should we check for hasOwnProperty instead?
- is this intentionally a different check from the one from nonTerminal?

Reply:
> is this intentionally a different check from the one from nonTerminal?

whoops!  not intentional